tests/core/contracts/test_contract_constructor.py

```diff
@@ -20,7 +20,7 @@ def test_contract_constructor_gas_estimate_no_constructor(MathContract):
     try:
         assert abs(gas_estimate - 167412) < 200  # Geth
     except AssertionError:
-        assert abs(gas_estimate - 200321) < 200  # eth-tester with py-evm
+        assert abs(gas_estimate - 177396) < 200  # eth-tester with py-evm
```

Comment:
An alternative to consider: set the actual gas cost on the left, and the 21000 tolerance on the right (instead of 200).

Reply:
@carver Replacing the hardcoded value with the actual gas usage is showing the gas estimation to be >21000 off in most cases.